iset.mm

```diff
@@ -64121,6 +64129,112 @@ all reals (greatest real number) for which all positive reals are greater.
+      XMBGZUCZUDZHIZAHIUAUBBAXKJHUEXKXMHUEXKXQHUEXKAHUEXLJKCGZUFZDGZXSLZJMZDJUG
+      ZYBNMZDJOZPZQZCJRIXLYHCUHUIUJDJCRUKULYGXTYFYDYEDUMUNUPUOXMVGIZXNXRYIXNTZX
+      QKEGZUFZFGZYKLZJMZFXQUGZYNNMZFXQOZPZQZEUHZXRYJYTEYJYLYSYJYLTZYAYKXMUQZLZJ
+      MZDXMUGZYSUUDNMZDXMOZUUBUUEYSDXMUUBYAXMIZTZUUEYSUUJUUETZYPYRUUKYAXQIZYAYK
+      LZJMZYPUUIUULUUBUUEYAXMXPURUSUUKUUDUUMJUUIUUDUUMMUUBUUEYAXMYKUTUSUUJUUEVA
+      VBYOUUNFYAXQYMYAMYNUUMJYMYAYKVCSVDVEVTVHVFUUBXOYKLZJMZUUHYSQUUONMZUUBUUPT
+      ZYSUUHUURYPYRUUBXOXQIZUUPYPUUSUUBXOXPIUUSBVIXOXPXMVJULUPZYOUUPFXOXQYMXOMZ
+      YNUUOJYMXOYKVCZSVDVKVTVLUUBUUQTZUUHYSUVCUUHTZYRYPUVDYQFXMOZYQFXPOZYRUVDUU
+      HUVEUVCUUHVAUUHYMUUCLZNMZFXMOUVEUUGUVHDFXMYAYMMUUDUVGNYAYMUUCVCSVMUVHYQFX
+      MYMXMIUVGYNNYMXMYKUTSVNVOVPUVDUUQUVFUUBUUQUUHVQYQUUQFXOBVRZUVAYNUUONUVBSV
+      SWAYQFXMXPWBVEWCVHUUBUUOJNWDZIUUPUUQPUUBUUOKUVJUUBXQKXOYKYJYLVAUUTWEWFWGU
+      UOJNWKWLWHYJXMKUUCUFZUUFUUHPZYLYJXMKXSUFZYCDXMUGZYEDXMOZPZQZCUHZUVKUVLQZX
+      NUVRYIXNUVRXMRIXNUVRUIUBVRZDXMCRUKULWIWJUVQUVSCUUCYKXMEVRWMXSUUCMZUVMUVKU
+      VPUVLXMKXSUUCWNUWAUVNUUFUVOUUHUWAYCUUEDXMUWAYBUUDJYAXSUUCWOZSWPUWAYEUUGDX
+      MUWAYBUUDNUWBSWQWRWSWTWLYLXMXQXAUVKXMXPXBXQKXMYKXCXDXEXFVHXGXQRIXRUUAUIXM
+      XPUVTXOUVIXHXIFXQERUKULWAVHXJ $.
+  $}
+
+  ${
+    $d f x y $.  $d ph x y z $.
+    omniexmid.1 $e |- x e. Omni $.
+    $( If all sets are omniscient, excluded middle follows.  (Contributed by
+       Jim Kingdon, 29-Jun-2022.) $)
+    omniexmid $p |- ( ph \/ -. ph ) $=
+      ( vy vz vf cv c0 cfv wceq wrex c1o wral wo eqeq1d c2o 0ex wi wcel ax-mp
+      csn crab cxp wn p0ex rabex xpeq1 fveq1d rexeqbi1dv raleqbi1dv orbi12d cpr
+      wf prid1 df2o3 eleqtrri fconst6 wal comni cvv vex dfomnicl mpbi xpex feq1
+      fveq1 rexbidv ralbidv imbi12d spcv vtocl rexm biidd elrab simprbi exlimiv
+      wb wex syl neii wa snid biantrur bitr4i fveq2 rspcv sylbi impcom fvconst2
+      1n0 adantl eqtr3d ex mtoi orim12i ) DGZAEHUAZUBZWQUCZIZHJZDWRKZWTLJZDWRMZ
+      NZAAUDZNWPBGZWQUCZIZHJZDXGKZXILJZDXGMZNZXEBWRAEWQUEUFXGWRJZXKXBXMXDXJXADX
+      GWRXOXIWTHXOWPXHWSXGWRWQUGUHZOUIXLXCDXGWRXOXIWTLXPOUJUKXGPXHUMZXNXGHPHHLU
+      LPHLQUNUOUPUQXGPFGZUMZWPXRIZHJZDXGKZXTLJZDXGMZNZRZFURZXQXNRZXGUSSZYGCXGUT
+      SYIYGVQBVAZDXGFUTVBTVCYFYHFXHXGWQYJUEVDXRXHJZXSXQYEXNXGPXRXHVEYKYBXKYDXMY
+      KYAXJDXGYKXTXIHWPXRXHVFZOVGYKYCXLDXGYKXTXILYLOVHUKVIVJTTVKXBAXDXFXBWPWRSZ
+      DVRAXADWRVLYMADYMWPWQSAAAEWPWQEGZWPJAVMVNVOVPVSXDALHJZLHWJVTXDAYOXDAWAHWS
+      IZLHAXDYPLJZAHWRSZXDYQRAHWQSZAWAYRYSAHQWBWCAAEHWQYNHJAVMVNWDZXCYQDHWRWPHJ
+      WTYPLWPHWSWEOWFWGWHAYPHJZXDAYRUUAYTWRHHQWIWGWKWLWMWNWOT $.
+  $}
+
+  ${
+    $d f x y $.
+    $( Given excluded middle, every set is omniscient.  Remark following
+       Definition 3.1 of [Pierik], p. 14.  Given ~ omniexmid it presumably
+       would be possible to strengthen this to a biconditional.  (Contributed
+       by Jim Kingdon, 29-Jun-2022.) $)
+    exmidomni $p |- ( EXMID -> A. x x e. Omni ) $=
```

Comment:
> |- ( EXMID -> _V = Omni )

has the advantage of being variable-free and of looking like the axiom of constructibility V = L.

Reply:
As for `_V = Omni` I don't know, it doesn't read as well to me but maybe I'm just not used to it or something.

As for the converse, thanks for that, I've formalized that proof. That seems like a mostly general procedure that probably could work for a lot of the existing *exmid theorems.

Reply:
So maybe a lemma like
```
|- ( EXMID <-> A. x ( x C_ 1o -> ( E. y y e. x \/ x = (/) ) )
```
will turn out to be useful ? (works for any set, not necessarily included in 1o).  Just an idea for the future.

Reply:
So maybe a lemma like
```
|- ( EXMID <-> A. x ( x C_ 1o -> ( E. y y e. x \/ x = (/) ) )
```
will turn out to be useful ? (works for any set, not necessarily included in 1o).  Just an idea for the future.